src/test/java/com/softserveinc/ita/ProductPageTest.java

```diff
@@ -58,4 +58,23 @@ public void verifyDiscountPrice() {
                 .as("Price with discount should be lower than old price")
                 .isLessThan(preDiscountPrice);
     }
+
+    @Test
+    public void verifyKitPrice() {
+        var kitIndex = 2;
+        var productPage =  homePage.
+                openCategory("Ноутбуки та комп’ютери")
+                .openSubCategory()
+                .openProduct()
+                .swithKitTo(kitIndex);
+
+        var expectedPrice = productPage.getMainKitProductPrice(kitIndex) + productPage.getAdditionalKitProductPrice(kitIndex);
```

Comment:
extract to variables:
var mainKitProductPrice = productPage.getMainKitProductPrice(kitIndex)
var additionalKitProductPrice = productPage.getAdditionalKitProductPrice(kitIndex)